New Emergency Shuttle: The Fleet

CrazyDrakkon opened the pull request:
# Document the changes in your pull request

(My first PR) Adds massive new "shuttle"/collection of shuttles, each representing a department, that are connected via quantum pads. Includes a brig and medical shuttle.

![image](https://github.com/user-attachments/assets/421fd11d-08a1-44a5-80b4-aec5c4ca7326)

# Why is this good for the game?

Adds a fun, high-end shuttle option. Alternative to the Luxury Shuttle that isn't ugly.

# Testing

Tested. It fits.

![image](https://github.com/user-attachments/assets/89b6e31a-bda3-4c87-9aa8-d86eddc42142)

![image](https://github.com/user-attachments/assets/9e5c907f-a6c3-4ef6-ad7a-abbd3790f881)

# Changelog

:cl:

rscadd: Added new emergency shuttle: The Fleet

/:cl:

CrazyDrakkon committed:
Create emergency_fleet.dmm

CrazyDrakkon committed:
Update emergency.dm

CrazyDrakkon committed:
Emergency.dm grammar fix

accessible without access => publicly accessible

AMyriad reviewed (changes requested):
Agreed this is a mess I love it
On a more serious note, before I take a closer look, can you show proof of testing of this thing docking at the station and at centcom?

CrazyDrakkon committed:
Map fixes and description update

Made actually functional portals and minor changes, updated description accordingly.

CrazyDrakkon committed:
Minor Map Changes

Signage, shutter buttons, airlock names

CrazyDrakkon committed:
Name Consistency Fixes

CrazyDrakkon committed:
Science Shuttle Mobs Transportation

CrazyDrakkon committed:
Removes bluespace bodybags

Mobs don't fit

AMyriad reviewed (changes requested):
Alright, so this is a pretty funny idea and I respect the effort you put into this, but I don't see it working as it is now. Balance concerns aside (there is a lot of space to run/hide in), the main problem is this thing is just too BIG.

I did some size estimates to check, and sadly yes this shuttle is too chonky to fit in a number of places, such as the syndicate hijack base and donutstation for example. I'm sure it fits on boxstation and centcom fine, but mind you those aren't the only maps in existence.

And since this'd be probably the biggest shuttle design we'd be adding, whatever maps we make from now on would have to abide by its size, meaning we'd be forced to keep the evac docks absolutely clear of anything in a large radius just so this beast could fit.

![image](https://github.com/user-attachments/assets/daffa9ea-d75f-47bd-9b3f-14a605145e86)
![image](https://github.com/user-attachments/assets/db213fd5-1e7a-409e-a6d8-ca648a633b4d)

CrazyDrakkon committed:
Emergency.dm shuttle name consistency

CrazyDrakkon committed:
Bluespace pet carriers, champagne, and tweaks

CrazyDrakkon committed:
Removes Pod Inner Doors

Removes closed 1x1 areas from pods
